shub/utils.py

```diff
@@ -100,7 +100,9 @@ def patch_sys_executable():
             try:
                 # Will raise NotFoundException if no Python installation found
                 py_exe = find_exe('python')
-                if '2.' not in get_cmd_output([py_exe, '--version']):
+                py_ver = subprocess.check_output([py_exe, '--version'],
+                                                 stderr=subprocess.STDOUT)
+                if '2.' not in py_ver:
```

Comment:
Just realized the current test also matches `3.2.0` :see_no_evil:
I'm fine with checking 2.7. I thought Ubuntu Precise was still on 2.6 but even they have upgraded. Why the `^Python`?